app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PostsController < ApplicationController
+  # load_and_authorize_resource
```

Comment:
- [ ] Great job. kindly uncomment this line for it is needed to use the CanCanCan for Post authorization in Post Controller

Reply:
i decided after testing that the uncommented line didn't serve any purpose for the page at the moment hence decided to deleted